examples/sos_decomposition.jl

```diff
@@ -0,0 +1,52 @@
+using DynamicPolynomials
+using SumOfSquares
+using JuMP
+using SCS
+
```

Comment:
Can you add your name here after `**Contributed by**: `? This is not done for other notebboks yet but I am reworking them at the moment